src/views/Count.vue

```diff
@@ -2,7 +2,7 @@
   <ion-page>
     <ion-header>
       <ion-toolbar>
-        <ion-title slot="start">{{ translate("Count name") }}</ion-title>
+        <ion-title slot="start">{{ currentFacility?.facilityName || translate("Count name") }}</ion-title>
```

Comment:
If facilityName is not present display facilityId, we do not need to display static text `Count name` in any case.

Reply:
Resolved in https://github.com/hotwax/inventory-count/pull/493